pwiz_tools/Skyline/Properties/Resources.resx

```diff
@@ -2802,6 +2802,9 @@ Are you sure you want to continue?</value>
   <data name="StaticMod_Validate_Loss_only_modifications_may_not_be_variable" xml:space="preserve">
     <value>Loss-only modifications may not be variable.</value>
   </data>
+  <data name="StaticMod_Validate_Loss_only_modifications_may_not_be_explicit" xml:space="preserve">
```

Comment:
I deleted this string in an earlier commit. You must have had a bad merge if it's showing up here.

Reply:
Yes, I have to manually resolve merge conflicts on Resources.resx pretty much every time I merge with master.
Removed the string.